tests/integration_tests.rs

```diff
@@ -87,6 +62,23 @@ fn test_model_graph_traversal() {
         output: Visibility::Public,
     };
 
+    let model_path = build_model_path("models/smol-llm.onnx");
```

Comment:
LLM model
I think we have to implement most of the Operators to make the proof system work.

Reply:
and ONNX file larger than 200MB is not suitable for the github repo file.
We should limit the binary file size or we should upload it to S3 bucket.

Reply:
> I think we have to implement most of the Operators to make the proof system work.

Great, if we can create operations for transformers, I think we can port most models.

> We should limit the binary file size or we should upload it to S3 bucket.

I'll use the DVC to improve it during refactoring, thanks!